rave-compiler/src/test/resources/fixtures/test2/MyFactory_Generated_Validator.java

```diff
@@ -35,6 +36,7 @@
     addSupportedClass(ArrayNotNull.class);
     addSupportedClass(AbstractAnnotated.class);
     addSupportedClass(IntDefModel.class);
+    addSupportedClass(LongDefModel.class);
```

Comment:
Alpha this like the rest. I know it's generated code, so you'll want to fix in the generation

Reply:
We're actually not alpha here, peak a little bit above :) Would have to add a sort by class name in the processor and switch all the tests.